tests/integration/test_fasttext_model_artifact.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+import tempfile
+import json
+
+import bentoml
+from tests.bento_service_examples.fasttext_classifier import FasttextClassifier
+from bentoml.yatai.client import YataiClient
+
+import fasttext
+
+
+@pytest.fixture()
+def fasttext_classifier_class():
+    FasttextClassifier._bento_service_bundle_path = None
+    FasttextClassifier._bento_service_bundle_version = None
+    return FasttextClassifier
+
+
+test_json = json.dumps({"text": "foo"})
+
+
+def test_fasttext_artifact_pack(fasttext_classifier_class):
+
+    temporary_file = tempfile.NamedTemporaryFile(suffix=".txt", mode="w+")
+    temporary_file.write("__label__bar foo")
+    # Set file pointer to begging to ensure correct read
+    temporary_file.seek(0)
```

Comment:
You can just close the temporary file here--no need for `seek`.

Reply:
The temporary file ends up being automatically destroyed once it is closed().
Would you suggest using a function such as [mkstemp](https://docs.python.org/3/library/tempfile.html#tempfile.mkstemp)? 
File deletion would have to be handled explicitly in that case!

Reply:
I lied! I assumed `tempfile` worked like `mkstemp`, but it doesn't. You did the right thing.